### ISSUE TITLE
webp support?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#610]
```
hi,

i know sumatra supports webp format (great space savings) since  prelease 8298 
(June 2013) 

are you going to support this format (inside cbr/cbz) in ebookdroid?

thanks for considering!!
```

Original issue reported on code.google.com by `ble...@gmail.com` on 15 Sep 2013 at 5:23

**Comment by GoogleCodeExporter** [comment]
```
Possibly in 2.1.0
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 15 Sep 2013 at 5:40
- Changed state: **Started**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Hi, are there any example of document from real world that uses this codec?
```

Original comment by `Andrei.K...@gmail.com` on 16 Sep 2013 at 7:07

**Comment by GoogleCodeExporter** [comment]
```
hi, 
yes, you could get more info here and one example of file:

https://code.google.com/p/sumatrapdf/issues/detail?id=2356

also i found here a converter: 
http://tomeko.net/software/CbxConverter/index.php?lang=en

i think webp could be a very nice addition in epubs and cbr/cbz, the space 
gains are very worth of it!
```

Original comment by `ble...@gmail.com` on 16 Sep 2013 at 7:10

**Comment by GoogleCodeExporter** [comment]
```
Implemented and will be available in 2.1.0.
The development version is available here:
https://drive.google.com/?tab=mo&authuser=0#folders/0B6DTNNagrvFPbU13UmxiekE2VDg
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 16 Sep 2013 at 2:14
- Changed state: **Done**
